plutus-tx-plugin/test/AssocMap/Spec.hs

```diff
@@ -91,21 +91,23 @@ map2 =
         \n ->
           let m1 =
                 Data.AssocMap.unsafeFromList
-                  [ (n PlutusTx.+ 1, "one")
```

Comment:
The fact that this test relies on UTF-8 encoding for BuiltinByteStrings is accidental

Reply:
I've re-worked the test to use `Integer`s instead of `BuiltinByteString`s.